### DISCUSSION TITLE
Piezo Crystals facing the wrong way

### DISCUSSION THREAD

**Discussion by GracedTarkov** [FakeFishGames/Barotrauma#15012] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Piezo Crystals always face upwards instead of outwards from the surface they are sticking to.

![Screenshot 2024-10-31 120538](https://github.com/user-attachments/assets/929940d9-a54a-49f5-a079-0e95a6caa11c)

### Reproduction steps

Host a multiplayer or start a singleplayer game, go to the Great Sea, look for Piezo Crystals.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by KougamiYuriko** [comment]
Yes, I've encountered this too. It looks quite funny, and I wonder when the developers will fix this small issue.

**Comment by Jasontti** [comment]
Tested a fix for this, no issues found.
Crystals and other environment detail spawn in right way now.
Closing discussion as resolved.
